demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -1623,3 +1624,27 @@ def verify_yml_commands_match_readme(self, is_modified=False):
                 is_valid = False
 
         return is_valid
+
+    @error_codes('IN153')
+    def verify_reputation_commands_has_reliability(self, is_modified: bool = False):
+        """
+        In case the integration has reputation command, ensure there is a reliability parameter.
+        Args:
+            is_modified (bool): Whether the given files are modified or not.
+
+        Return:
+            bool: True if there are no reputation commands or there is a reliability parameter
+             and False if there is at least one reputation command without a reliability parameter in the configuration.
+        """
+        if not is_modified:
+            return True
+        commands_names = [command.get('name') for command in self.current_file.get('script', {}).get('commands', [])]
+        yml_config_names = ' '.join([config.get('name') for config in self.current_file.get('configuration', {})])
+        for command in commands_names:
+            if command in REPUTATION_COMMAND_NAMES:
+                if re.search('reliability', yml_config_names.lower()):
```

Comment:
it's ok but fyi, better to avoid using `re` when possible (for complexity reasons), e.g. `'reliability' in yml_config_names.lower()` would be great here too.